src/clients/index.ts

```diff
@@ -0,0 +1,7 @@
+// Interfaces
+export { SearchClient } from './SearchClient'
+export { TrendingClient } from './TrendingClient';
+
```

Comment:
What about exporting a "full client" type as well? In the [tests](https://github.com/jamescallumyoung/gif-provider/pull/17/files#diff-6ebac67d754e8cdebc1299260e19feac2b9da3afc26790a51b4b4837e7f890c3R22) there is a similar type:

```ts
type ClientType = SearchClient & TrendingClient;
```